x/gamm/types/events.go

```diff
@@ -1,11 +1,5 @@
 package types
 
-import (
-	"strconv"
-
-	sdk "github.com/cosmos/cosmos-sdk/types"
-)
-
 const (
```

Comment:
Consider moving these and replacing these type calls with the newly added events emit. IMO could be confusing to maintain two different event packages for a single module.

Reply:
The motivation behind leaving constants in types was to have them exposed / exported to clients who might want to use these constants to subscribe to events.

However, the emission logic should only be exposed to the gamm package, not the clients. That's why it was moved to an internal package inside gamm.

This is to preserve information hiding where we only export what clients should be able to see while unexport any package-private functionality.

Please let me know what you think

Reply:
Let's continue discussing this in this thread please:
https://github.com/osmosis-labs/osmosis/pull/1983#discussion_r915894294